src/game/scoring.rs

```diff
@@ -3,11 +3,14 @@ use arrayvec::IntoIter;
 use itertools::*;
 use std::convert::TryInto;
 
-pub fn score_hand(hand: &[Card], shared_card: &Card, is_crib: bool) -> i32 {
+pub fn score_hand(hand: &[Card], shared_card: Option<&Card>, is_crib: bool) -> i32 {
```

Comment:
Why is shared_card an Option? Which scenario is this supporting?

Reply:
This was a pull over from CribbageJS. May not be needed after full implementation, but looked like a short circuit when calling from select_card.rs.